oat-sa/extension-tao-itemqti
# Feature/CON-33/passage scrolling support new ui

Related to: https://oat-sa.atlassian.net/browse/CON-33
Related to: https://github.com/oat-sa/tao-test-runner-qti-fe/pull/335

Fixed issue with Save - update `container.body` after Active Include widget exit
Added support for new UI test runner - added classes `tao-overflow-y` for height `tao-full-height` and other

How to test:

1. create passage
2. create item with this passage
3. enable scroll for passage and save item
4. go to test Preview and check that passage is scrollable
